server/server_test.go

```diff
@@ -207,6 +207,7 @@ func runTestPreparedString(t *C) {
 func runTestLoadData(c *C) {
 	path := "/tmp/load_data_test.csv"
 	runTests(c, dsn+"&allowAllFiles=true", func(dbt *DBTest) {
```

Comment:
try to use "&allowAllFiles=true" to control client's capability and avoid mockLoadDataClientLocalFilesCap